tuning/config/configs.py

```diff
@@ -34,8 +34,8 @@
 class ModelArguments:
     model_name_or_path: Optional[str] = field(default="facebook/opt-125m")
     use_flash_attn: bool = field(
-        default=True,
-        metadata={"help": "Use Flash attention v2 from transformers, default is True"},
+        default=False,
```

Comment:
for library users would have been good to leave it as True, we need the system testing team to keep it True. 

can we set it to False temporarily in launch_training script instead until we figure out what the issue was?

Reply:
thanks

Reply:
you are yet to commit right?